platform.json

```diff
@@ -53,7 +53,14 @@
       "homepage": "https://github.com/OpenAtomFoundation/TencentOS-tiny",
       "title": "TencentOS Tiny"
     },
-    "arduino": {
+    "arduino-openwch": {
```

Comment:
Addition of a new framework identifier should not be needed, PlatformIO does core selection too via `board_build.core` (https://docs.platformio.org/en/latest/platforms/ststm32.html#switching-between-arduino-cores).